docker-compose-services/sqlsrv/Dockerfile

```diff
@@ -1,5 +1,4 @@
-ARG BASE_IMAGE
-FROM $BASE_IMAGE
+ARG odbc_version=2.3.7
```

Comment:
Shouldn't this be ENV? I don't see where you're using it as an arg. Here you're using it as an argument with a default value.

Reply:
I used ARG instead of ENV because I need this variable in image building only (not in the container environment). I use it in rows from 15 to 18 

```
...
    unixodbc=$odbc_version \
    odbcinst=$odbc_version \
    odbcinst1debian2=$odbc_version \
    unixodbc-dev=$odbc_version \
...
```

Reply:
I can go with that :)